Tasks/DownloadBuildArtifacts/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,20 +1,31 @@
 {
-  "loc.friendlyName": "Download Artifacts - Build",
+  "loc.friendlyName": "Download Build Artifacts",
   "loc.helpMarkDown": "",
-  "loc.description": "Download build artifacts",
-  "loc.instanceNameFormat": "Download Artifacts - Build",
+  "loc.description": "Download Build Artifacts",
+  "loc.instanceNameFormat": "Download Build Artifacts",
+  "loc.input.label.buildType": "Download artifacts produced by",
+  "loc.input.help.buildType": "Download artifacts produced by the current build, or from a specific build.",
   "loc.input.label.project": "Project",
-  "loc.input.help.project": "Select the project name",
+  "loc.input.help.project": "The project from which to download the build artifacts",
   "loc.input.label.definition": "Build definition",
   "loc.input.help.definition": "Select the build definition name",
-  "loc.input.label.version": "Build",
-  "loc.input.help.version": "Select the build number",
-  "loc.input.label.itemPattern": "Item Pattern",
-  "loc.input.help.itemPattern": "Minmatch pattern to filter files to be downloaded.",
+  "loc.input.label.buildId": "Build",
+  "loc.input.help.buildId": "The build from which to download the artifacts",
+  "loc.input.label.downloadType": "Download Type",
+  "loc.input.help.downloadType": "Download specific or multiple artifacts matching pattern.",
+  "loc.input.label.artifactName": "Artifact name",
+  "loc.input.help.artifactName": "The name of the artifact to download",
+  "loc.input.label.itemPattern": "Matching Pattern",
+  "loc.input.help.itemPattern": "Minimatch pattern to filter files to be downloaded. To download all files within artifact drop use drop/**",
   "loc.input.label.downloadPath": "Destination directory",
   "loc.input.help.downloadPath": "Path on the agent machine where the artifacts will be downloaded",
   "loc.messages.DownloadArtifacts": "Downloading artifacts from: %s",
   "loc.messages.LinkedArtifactCount": "Linked artifacts count:  %s",
   "loc.messages.FileContainerInvalidArtifactData": "Invalid file container artifact. Resource data must be in the format #/{container id}/path",
-  "loc.messages.UnsupportedArtifactType": "Unsupported artifact type: %s"
+  "loc.messages.UnsupportedArtifactType": "Unsupported artifact type: %s",
+  "loc.messages.BuildNotFound": "Build with id: %s not found",
```

Comment:
Can we be consistent about whether we use a colon after id?  We are all over the place here.
```
Build with id: 3 not found
Build with id 3 not found for build definition id 2
```

Reply:
removed the colon